ui/src/containers/HeaderContainer.js

```diff
@@ -56,8 +56,8 @@ HeaderContainer.propTypes = {
 
 function mapStateToProps (state) {
   return {
-    currentUser: state.header.user,
-    isUserRequestFailed: state.header.isUserRequestFailed
+    currentUser: state.user.user,
```

Comment:
Let's remove the outer user object in the reducer so we don't have the nested objects here. We're doing that in the other reducer states.

Reply:
Like what? user.currentUser?